.github/workflows/ci.yml

```diff
@@ -254,6 +254,9 @@ jobs:
           version: ${{ env.DOCKER_BUILDX_VERSION }}
           install: true
 
+      - name: install skopeo
+        run: sudo apt-get -y install skopeo
+
```

Comment:
We usually install dependencies as part of the make target, see https://github.com/upbound/build/blob/a81d4729219d410567f848d10ac6798d9862e398/makelib/golang.mk#L301 or https://github.com/upbound/build/blob/a81d4729219d410567f848d10ac6798d9862e398/makelib/k8s_tools.mk#L146, helm and kind are then just linked for the e2es in the `e2e-run-tests` target.
But if possible I'd rather avoid adding one more tool.

Reply:
I tried to avoid using `skopeo`, was thinking to use `kubectl port-forwarding`, and it did work on my laptop, but not on github, see previous runs: https://github.com/crossplane/crossplane/actions/runs/5427726428/jobs/9871266837#step:12:476  Looking around I found https://github.com/kubernetes/kubectl/issues/1169 - the issue is probably connected to kubectl version, but it would be really hard to debug it.

Unfortunately, we need to use https://github.com/containers/skopeo but it cannot be installed like helm or kind, it is distributed only through packages. BTW, I discovered that `scopeo` is already available in that ubuntu image and we could actually skip this step, like we do not install `kubectl` at all.

The reason for using `skopeo` is the test setup we need to achieve for testing the new functionality end-to-end:
* we need to deploy a registry with custom TLS certificate, with dns name that could be later accessible by crossplane
* we need to copy fn image to such registry from host where it got built. Such copy operation needs to ignore TLS certificate and establish the connection anyway. `docker push` does not have such flag, it would require to reconfigure the docker engine daemon, and such operation is even less portable between operating systems and docker installations.
* Hence it was needed to take `skopeo` - actually is a great tool for talking to registries and more.

All above was the reason why I initially started with the integration tests - they were running against local docker engine and in that case docker can push images without checking certificate. Also, the test setup was much simpler.

Reply:
But we need to run it also locally. Maybe the pull/push could be done using go-containerregistry directly with code?